api_test.go

```
@@ -39,7 +39,7 @@ func TestDumpHTTPLargeBodyDoesNotAllocate(t *testing.T) {
 	var m0, m1 runtime.MemStats
 	runtime.GC()
 	runtime.ReadMemStats(&m0)
-	if err := c.dumpHTTP(req, resp); err != nil {
+	if err := c.dumpHTTP(req, resp,nil); err != nil {
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofumpt)